fv3core/stencils/fv_dynamics.py

```diff
@@ -169,10 +169,19 @@ def do_dyn(state, comm):
         domain=grid.domain_shape_standard(),
     )
     print("DynCore", grid.rank)
+    if timer is not None:
+        # start dyn timer
```

Comment:
Please avoid comments saying what the code is doing, in favor of making the code clear enough that you can tell what it is doing. In this case, your code is already clear, and I don't think these comments are needed.